build.gradle

```diff
@@ -220,6 +226,18 @@ subprojects {
   tasks.withType(ScalaCompile) {
     scalaCompileOptions.useAnt = false
 
+    scalaCompileOptions.additionalParameters = [
+      "-deprecation",
+      "-unchecked",
+      "-encoding", "utf8",
+      "-target:jvm-1.7",
+      "-Xlog-reflective-calls",
+      "-feature",
+      "-language:postfixOps",
+      "-language:implicitConversions",
```

Comment:
I'm also ok with the import, but it is complete 180 turn on what we did before (only import the converters in the smallest possible scope to avoid any accidental conversion or something).

Perhaps a community discussion on the list about reversing this convention? I'm not suggesting a vote, but an announcement and give people chance to object if they feel strongly about it?